environment/env.go

```diff
@@ -37,12 +37,15 @@ const (
 	// CassandraDefaultPort Cassandra default port
 	CassandraDefaultPort = "9042"
 
+	// which sql system to use
+	SQLDriver        = "SQL_DRIVER"
+	SqlDefaultDriver = "postgres"
```

Comment:
for compatibility, I guess `SqlDefaultDriver = "mysql"` unless overrided
this is because there are existing customers who are already using mysql